### ISSUE TITLE
Video title looks like a heading but isn't coded as such

### ISSUE THREAD

**Issue by Woebin** [Chocobozzz/PeerTube#7137]
### Describe the current behavior

This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG22) [SC 1.3.1 (A) Info and relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships).

Marking up headings with the correct HTML heading elements allows accessibility features to work properly, and users can list and jump between headings smoothly.

The video title visually looks like a heading and serves the function of a heading in relation to the video contents, but it isn't coded as one.
![Video title looks like a heading, but is actually a div.](https://github.com/user-attachments/assets/3a554cdd-1b35-49d0-919f-3c47f25ed26f)

As an aside, these `<div>` elements have redundant `title` attributes that just repeat the visible text. These `title` attributes add nothing of apparent value and can likely be removed.

### Steps to reproduce

1. Inspect the video heading using the browser's developer tools.
2. Notice that it isn't an `<h*>` element as expected.

OR

1. Navigate from heading to heading using assistive technology, for example a screen reader.
2. Fail to find the video title as it's not a proper heading.

### Describe the expected behavior

The video title visually functions as a heading - to sighted users, the relationship between video title and video content is clear. The same relationship should be clear to users of assistive technology, which requires it to be programmatically determinable.

In this case, as the video is embedded in another page that should have its own main heading, it makes sense to use `<h2>` for the video title.

**Code example:**
```html
<h2 class="peertube-dock-title">Live stream video player test for Axess Lab</h2>
<span class="peertube-dock-description">Watching this video may reveal your IP address to others.</span>
```

### Additional information

* PeerTube instance:
  * URL: https://peertube-eu.howlround.com/
  * Version: 7.2.1
  * NodeJS version: Unknown
  * Ffmpeg version: Unknown

* Browser name, version and platforms on which you could reproduce the bug: MacOS 15.5, Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).

**Comment by Chocobozzz** [comment]
Thank you for the issue! Should be fixed by https://github.com/Chocobozzz/PeerTube/commit/d2064d873bc434ada862960fee3f99644681ea39

**Comment by Woebin** [comment]
Love to see such a quick turnaround on a11y issues! You're a star!
